libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -308,42 +306,27 @@ internal class MapboxTripSession(
                 return
             }
 
-            updateRouteProgressJob?.cancel()
-            updateRouteProgressJob = mainJobController.scope.launch(Dispatchers.Main.immediate) {
```

Comment:
Is it OK that you removed updateRouteProgressJob assignment here? Now it is only cancelled, never assigned. We should either remove it completely or return the assignment.

Reply:
Great catch! This went unnoticed, we definitely need to remove `updateRouteProgressJob` altogether as it's not used anymore. @VysotskiVadim could you cut a quick follow up PR when you have a chance? 🙏

Reply:
@dzinad , thank you! https://github.com/mapbox/mapbox-navigation-android/pull/6136